src/components/shared/select/Select.tsx

```diff
@@ -0,0 +1,24 @@
+import "./Select.scss";
+
+export interface SelectProps {
+  selectName: string;
+  data: string[];
+}
+
+const Select = ({ selectName, data }: SelectProps) => {
+  return (
+    <div className="select">
+      <label>{selectName}</label>
```

Comment:
are you sure that it should be a label?

Reply:
I believe so. Label over select exists on the project.